pkg/parquet/writer.go

```diff
@@ -185,3 +193,37 @@ func makeTags(tags map[string]string) []*s3.Tag {
 
 	return s3Tags
 }
+
+func (w *s3Writer) DeleteReadFiles(ctx context.Context) error {
```

Comment:
DeleteReadFiles and DeleteWrittenFiles look quite similar, can we reuse something?